electroncash_gui/qt/main_window.py

```diff
@@ -38,6 +38,7 @@
 from collections import OrderedDict
 from typing import List, Optional
 
+from PyQt5 import QtWidgets
```

Comment:
Note that this is needed because in Electrum ABC I got rid of all the `from ... import *`.
Alternatively you could have just remove the `QtWidgets` namespace from all occurences, as the entire module is imported 3 lines below.

e.g. `QtWidgets.QFileDialog` --> `QFileDialog`

Reply:
Done in https://github.com/scinklja/Electron-Cash/commit/2eca8153ca5e0454257ef72be771bf213f45384f